### ISSUE TITLE
How to deploy My APP/server folder to heroku

### ISSUE THREAD

**Issue by AndeYashwanth** [Rajpra786/STET-Exam-Management-System#1]
I see that you deployed MY APP/server folder to heroku at https://stet2020.herokuapp.com/
Can you tell me what steps did you follow to deploy subfolder in heroku?

Also, I faced a lot of errors when running the project in local. 

1. url paths in code are `/api/...`  but the folder structure doesn't contain`/api/`
2. some config and env files are missing
3. so many **implementations** are missing in app level **build.gradle** file
4. Mongodb url's are incomplete/different/incorrect
5. details regarding firebase project creation, enabling authentication, adding **sha1** and **sha256**, enabling devicecheckapi in cloud console are missing in readme(I spent almost a day figuring out these).
6. Could've made deployement easier by including all commands in a bat/sh file and running it
7. Details about running `npm run build` in client folders is missing in Readme files

**Comment by Rajpra786** [comment]
Yes you are right, the project is missing some code files. We will update all these files soon.

**Comment by AndeYashwanth** [comment]
Can you please update the repo sooner if possible?
